Tests/Fixtures/project/app/AppKernel.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+class AppKernel extends Kernel
```

Comment:
Not applicable here either, this is AppKernel, it is usually given without namespace, and this is part of fixtures...